src/pages/admin/account-management.tsx

```diff
@@ -0,0 +1,153 @@
+import DeleteModal from "@/components/account-management/DeleteModal";
+import { Button } from "@/components/ui/button";
+import CrossIcon from "@/components/ui/icons/crossicon";
+import WarningIcon from "@/components/ui/icons/warningicon";
+import { IAdmin } from "@/server/db/models/AdminModel";
+import { UNDELETABLE_EMAILS } from "@/utils/consts";
+import { Box, Tooltip } from "@chakra-ui/react";
+import { ObjectId } from "mongodb";
+import { useEffect, useState } from "react";
+
+export type Admin = IAdmin & { _id: ObjectId };
+
+const AccountManagementPage = () => {
+  const [admins, setAdmins] = useState<Admin[]>([]);
+  const [newEmail, setNewEmail] = useState<string>("");
+  const [emailError, setEmailError] = useState<string>("");
+  const [deleteModalDisclosure, setDeleteModalDisclosure] = useState(false);
+  const [selectedDeleteAdmin, setSelectedDeleteAdmin] = useState<Admin | null>(
+    null,
+  );
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const promise = await fetch("../api/admin");
+        const data = await promise.json();
+        const removableAdmins = data.filter(
+          (admin: Admin) => !UNDELETABLE_EMAILS.includes(admin.email),
+        );
+        setAdmins(removableAdmins);
+      } catch (e) {
+        console.log(e);
+      }
+    };
+    fetchData();
+  }, [admins]);
+
+  const handleAddAccount = async () => {
+    try {
+      const response = await fetch("../api/admin", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({ email: newEmail }),
+      });
+      if (response.ok) {
+        setNewEmail("");
+      } else {
+        const error = await response.text();
+        setEmailError(error);
+      }
+    } catch (error) {
+      console.log("Error adding account:", error);
+    }
+  };
+
+  return (
+    <div className="mb-28 mt-10">
+      <div className="text-center text-5xl font-semibold text-black">
+        Account Management
+      </div>
+      <div className="mt-24 flex flex-col px-28">
+        <div className="text-2xl font-semibold text-black">Add New Account</div>
+        <div className="mt-5 flex flex-row flex-wrap justify-between gap-5">
+          <input
+            className={`h-16 w-full max-w-[57rem] rounded-xl border bg-gray-50 px-7 text-lg ${
+              emailError ? "border-red-600" : "border-stone-500"
+            }`}
+            placeholder="Email"
+            value={newEmail}
+            onChange={(e) => setNewEmail(e.target.value)}
+            onClick={() => {
+              if (emailError != "") {
+                setEmailError("");
+              }
+            }}
```

Comment:
Instead of clearing the error on click, can the onClick function here be removed and then use `setEmailError("");` in `handleAddAccount`?